docs/open-source-projects.md

```diff
@@ -45,41 +53,67 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
-  style={{ width: "30%" }}
-/>
+> 
+<p align="left">
+	<a href="https://eosrate.io">
+		<img src="https://raw.githubusercontent.com/eoscostarica/design-assets/master/logos/eosrate/eosrate--horizontal-solid-transparent-overlight.png" width="30%"/>
+	</a>
+</p>
 
-[EOS Rate](https://eosrate.io) is an open-source dApp that allows EOS token holders to rate Block Producers (BPs) in just a few clicks. Users can access a rating system and voting portal, making it easier for token holders to make an informed decision.
+> EOS Rate is an open-source dApp that allows EOS token holders to rate Block Producers (BPs) in just a few clicks. Users can access a rating system and voting portal, making it easier for token holders to make an informed decision.
 
-- **Website** : https://eosrate.io
+- **Website** : [EOSIO Rate](https://eosrate.io)
 
-- **GitHub** : https://github.com/eoscostarica/eos-rate
+- **GitHub** : [<img src="https://cdn-icons-png.flaticon.com/32/25/25231.png"/>]( https://github.com/eoscostarica/eos-rate)
 
-![EOS Rate](https://raw.githubusercontent.com/eoscostarica/guide.eoscostarica.io/master/static/img/OSS_screnshots/EOS_Rate.PNG)
 
-![EOS Rate](https://raw.githubusercontent.com/eoscostarica/guide.eoscostarica.io/master/static/img/OSS_screnshots/EOS_Rate_2.PNG)
+###  EOS Rate Page
+ 
+ <p align="center">
+	<a href="https://eoscostarica.io/">
+		<img src={useBaseUrl('img/OSS_screnshots/EOS_Rate.PNG')} width="100%"/>
+	</a>
+</p>
+
+<p align="center">
+	<a href="https://eoscostarica.io/">
+		<img src={useBaseUrl('img/OSS_screnshots/EOS_Rate_2.PNG')} width="100%"/>
+	</a>
+</p>
+
 
 * * *
 
-## LACChain EOSIO Testnet
+### LACChain EOSIO Testnet
 
-<img
-  src="https://raw.githubusercontent.com/lacchain/eosio.lacchain.net/master/static/img/logos/lacchain-eosio-logo.png"
-  style={{ width: "30%" }}
-/>
+> 
+<p align="left">
+	<a href="https://eosio.lacchain.net/en/">
+		<img src="https://www.lacchain.net/assets/images/logo-lacchain-footer.png" width="30%"/>
+	</a>
+</p>
 
-We are part of LACChain EOSIO, an open-source initiative that offers an EOSIO blockchain network that integrates into LACChain. 
+> We are part of LACChain EOSIO, an open-source initiative that offers an EOSIO blockchain network that integrates into LACChain. 
 
-- **Website** : https://eosio.lacchain.net/en/
+- **Website** : [LACChain EOSIO](https://eosio.lacchain.net/en/)
 
-- **GitHub** : https://github.com/lacchain/eosio.lacchain.net
+ - **GitHub** : [<img src="https://cdn-icons-png.flaticon.com/32/25/25231.png"/>](https://github.com/lacchain/eosio.lacchain.net)
 
-![LACChain EOSIO](https://raw.githubusercontent.com/eoscostarica/guide.eoscostarica.io/master/static/img/OSS_screnshots/lacchain-eosio-homepage.png)
+ ##  LACChain EOSIO Page
+ 
+ <p align="center">
+	<a href="https://eoscostarica.io/">
+		<img src={useBaseUrl('img/OSS_screnshots/lacchain-eosio-homepage.PNG')} width="100%"/>
```

Comment:
Would be better if you use a screenshot from https://eosio.lacchain.net/en/

Reply:
I'm a little confused, the links are the same for both comments.

https://github.com/eoscostarica/guide.eoscostarica.io/pull/399#discussion_r782368085
https://github.com/eoscostarica/guide.eoscostarica.io/pull/399#discussion_r782366666

@leisterfrancisco